src/styles/main.scss

```diff
@@ -0,0 +1,4 @@
+@import "./utils/variables";
+@import "./utils/mixins";
+@import "./block/clock.scss";
+@import "./reset.scss";
```

Comment:
Consider changing the order of imports. Import 'reset.scss' before other SCSS files to avoid overriding styles.

Reply:
i'm fix please check my work))